components/fxa-client/src/internal/mod.rs

```diff
@@ -0,0 +1,560 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Careful, this large diff is not rendered by default! :-)

Git thinks this is a new file, but it's actually the previous contents of `./src/lib.rs`, moved to `./src/internal/mod.rs` and only very lightly changed.

Reply:
phew :)